adam/experiment/log_experiment.py

```diff
@@ -90,6 +90,8 @@ def log_experiment_entry_point(params: Parameters) -> None:
         log_hypotheses_every_n_examples=params.integer(
             "log_hypothesis_every_n_steps", default=250
         ),
+        learner_logging_path=params.optional_creatable_directory("experiment_group_dir"),
+        starting_point=params.integer("starting_point", default=-1),
```

Comment:
It looks like this is missing control over disabling logging the learner state and providing a path to load a learner from.